schedule/yam/test_cases/cryptlvm+cancel_existing.yaml

```diff
@@ -11,11 +11,16 @@ vars:
 schedule:
   - installation/bootloader_start
   - installation/setup_libyui
-  - installation/product_selection/install_SLES
+  - '{{install_sles}}'
   - installation/licensing/accept_license
   - installation/registration/register_via_scc
   - installation/module_registration/register_module_desktop
   - installation/system_probing/cancel_encrypted_volume
   - console/validate_encrypted_partition_not_activated
+conditional_schedule:
```

Comment:
Some ideas that probably were not clear in the ticket:
- we don't practice conditional schedule, we are trying to solve the issue via yaml files based on default.
- /test_cases/ folder is our pristine folder for the successful unification, if the unification is partial we don't locate the schedule schedule until further success (perhaps a code refactor is needed) but all the schedule in this folder should use keys, be based on defaults.